src/BloomBrowserUI/react_components/bloomAvatar.tsx

```diff
@@ -19,6 +19,15 @@ export const BloomAvatar: React.FunctionComponent<{
     const borderStyle = props.borderColor
         ? `${borderSizeInt}px solid ${props.borderColor}`
         : undefined;
+
+    // react-avatar does not cache actual avatars. It only caches which gravatar urls failed
+    // (whether because user was offline or doesn't have a gravatar),
+    // and then doesn't retry the failed urls so long as they are valid in cache.
+    // We do want it to retry retrieving gravatars, so keep its cache empty
+    const cache = new Cache({
+        sourceTTL: 0, // retain for 0 milliseconds
```

Comment:
Does updating bloom clear the local storage? If not:
I did not give the cache a new prefix. This means that it if a user has failed gravatar urls in the cache, it may take up to a week for those cached urls to expire and therefore for this change to be visible. I thought this was better than the alternative, in which react-avatar starts using a cache at a new prefix and never clears whatever was in the default prefix cache.

But for now, to test this PR, you will need to clear local storage